sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/extensions/PlaybackOnlyUntil.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common.test.shared.extensions;
+
+import org.spockframework.runtime.extension.ExtensionAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.time.OffsetDateTime;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@ExtensionAnnotation(PlaybackOnlyUntilExtension.class)
+public @interface PlaybackOnlyUntil {
+    String expiryTime();
+}
```

Comment:
instead of creating new annotation could we just add optional parameter to existing PlaybackOnly ?